client/packages/common/src/ui/forms/JsonForms/api/hooks/document/useDocument.ts

```diff
@@ -1,8 +1,8 @@
 import { useQuery } from '@openmsupply-client/common';
-import { useDocumentApi } from '../utils/useDocumentApi';
+import { usePatientDocumentApi } from '../utils/useDocumentApi';
 
 export const useDocument = (name: string) => {
-  const api = useDocumentApi();
+  const api = usePatientDocumentApi();
```

Comment:
Does this need to have a patient-specific name? Thought it should be type-agnostic since its used for querying the document endpoint.

Reply:
Agree. The reason I did it was that the react query base key for patient mutations (insertPatient, updatePatient) is 'patient' and its also used for querying the patient list. So I named it usePatientDocumentApi to make it a bit clearer. But yes, would be good to get i more generic again. Any suggestions?